app/javascript/components/Product.jsx

```diff
@@ -226,3 +230,53 @@ const CommentInfo = ({ product }) => {
     </>
   )
 }
+
+const TrainingEndInfo = ({ product }) => {
```

Comment:
[IMO]関数名が少し広義的かもしれません。以下などどうでしょうか。👀
```suggestion
const TrainingEndDateInfo = ({ product }) => {
```